projects/robots/kinematics/tinkerbots/protos/TinkerbotsBrickAdapter.proto

```diff
@@ -21,7 +21,7 @@ PROTO TinkerbotsBrickAdapter [
           rotation 0 1 0 1.57
           children [
             Slot {
-              type "lego cross female"
+              type "lego cross-"
               endPoint IS slot
```

Comment:
Is it correct that here is `-`, what does it refer to?
I see two male plugs behind the block.

Note that `slot` is not defined in the PROTO header.

Reply:
You're right, this slot should be dropped.